yarn.lock

```diff
@@ -109,32 +109,43 @@
     monaco-editor "0.44.0"
     vscode "npm:@codingame/monaco-vscode-api@1.83.5"
 
-"@eclipse-glsp/client@1.1.0-RC10":
-  version "1.1.0-RC10"
-  resolved "https://registry.yarnpkg.com/@eclipse-glsp/client/-/client-1.1.0-RC10.tgz#c32c91f5c1410f32c97bf506ecf9b97c7dafeb20"
-  integrity sha512-vMdUu5X8waKNXGRU9DEylvTkMfayiCtWRy4e+TWygfiiM7G7wD8AZPL4fGQoDT3h7ulsgpXV+Mcorysp86P6lw==
+"@eclipse-glsp/client@2.1.0":
+  version "2.1.0"
+  resolved "https://registry.yarnpkg.com/@eclipse-glsp/client/-/client-2.1.0.tgz#324c48e243cb24d454c9c4a38dcd7effea92b729"
+  integrity sha512-OxQhgcz1VJnmj4IYMSz7Xp4bk50yX/nvrcdVyBvnWJ0S2FF9rJZcdMykooMg19N0nfnG+cjzVa4lbd5Mf975XQ==
   dependencies:
-    "@eclipse-glsp/protocol" "1.1.0-RC10"
-    autocompleter "5.1.0"
+    "@eclipse-glsp/sprotty" "2.1.0"
+    autocompleter "^9.1.0"
+    file-saver "^2.0.5"
     lodash "4.17.21"
-    sprotty "0.14.0-next.02bbac0.26"
 
-"@eclipse-glsp/ide@1.1.0-RC10":
-  version "1.1.0-RC10"
-  resolved "https://registry.yarnpkg.com/@eclipse-glsp/ide/-/ide-1.1.0-RC10.tgz#70e58c958c0b8487daa6489c368c53a7bb3a56d9"
-  integrity sha512-JBOqoto+IGK7v9JxL9MDJag0ir1VibmzC5ZfaE3YwkAucaN0frXm0Rrz5SJWTYsezJNu6eF7WdixYWc3C9ydAw==
+"@eclipse-glsp/ide@2.1.0":
+  version "2.1.0"
+  resolved "https://registry.yarnpkg.com/@eclipse-glsp/ide/-/ide-2.1.0.tgz#f1bc0fb76ab70f95aaad45bd39d5bef61c6d3a8e"
+  integrity sha512-2feg3u2jIbtKI6cO//vwAXRpsH5FeZFHKnL+7f6cKJw2yhEig9UCvwghR7ugwdmW9YWcMsVfw57rARHmmpjT7g==
   dependencies:
-    "@eclipse-glsp/client" "1.1.0-RC10"
+    "@eclipse-glsp/client" "2.1.0"
 
-"@eclipse-glsp/protocol@1.1.0-RC10":
-  version "1.1.0-RC10"
-  resolved "https://registry.yarnpkg.com/@eclipse-glsp/protocol/-/protocol-1.1.0-RC10.tgz#123fc350277431ce36a719b582c0cc12746df6a9"
-  integrity sha512-00sT68AbyiiMF63kXdQcQz0vPpf4Nuf1pWoxsJq82npyCIL1PP2rqqE3v+RfQQFic0cmYQObXhJCueK/BjvDaw==
+"@eclipse-glsp/protocol@2.1.0":
+  version "2.1.0"
+  resolved "https://registry.yarnpkg.com/@eclipse-glsp/protocol/-/protocol-2.1.0.tgz#e76a65a4815c969bfb86a1ab202ad24384a317cc"
+  integrity sha512-CUcVbsjiddDWEuhWKDLgfqslmau2QT6QTF9LyA4/y6Eomv5s8nTexeBMI6hCWL4sHgMfQ4t2zI9zV2a5gIow4A==
   dependencies:
-    sprotty-protocol "0.14.0-next.02bbac0.26"
+    sprotty-protocol "0.15.0-next.044bba2.13"
```

Comment:
Maybe a glsp problem? That the protocol package still use the sprotty-protocol v0.15.0?

Reply:
There is indeed a wrong setting within GLSP but it didn't cause any problems but glad you discovered it! I fixed the version now to 1.0.0 for sprotty-protocol.